tests/test_core.py

```diff
@@ -47,6 +49,8 @@ def decorated(self):
 def bleeding_edge_wasm_backend(f):
   def decorated(self):
     if not V8_ENGINE or V8_ENGINE not in JS_ENGINES:
+      if os.getenv('EMTEST_NO_D8') is None:
+        raise Exception('d8 is needed to run SIMD tests, set EMTEST_NO_D8 if you want to skip them')
```

Comment:
I think this might be a little more readable if the logic were inverted and the check was at the very top of the function:

```
if os.getenv('EMTEST_NO_D8'):
    skip()
```